Sometimes app crashes/stucks on Keys saved step during onboarding

This issue is caught by e2e from time to time. So far I was unable to reproduce manually. @churik has faced similar issue on IOS some time ago.

Link to the e2e steps video https://app.eu-central-1.saucelabs.com/tests/b0e4792562ca403a89a7b6296953cea1?auth=6cd3d5f3fa68bc9308cb71249fb1bdc0#23

[logcat (9).log](https://github.com/status-im/status-mobile/files/15495702/logcat.9.log)

I see panic in logcat

```
05-29 14:18:51.300  5387     0 E Go      : panic: runtime error: invalid memory address or nil pointer dereference
05-29 14:18:51.300  5387     0 E Go      : [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x73683a616540]
```

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

**_WAKU.GO_**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x73683a616540]

goroutine 1276 [running]:
github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/p2p/enode.(*Node).Seq(0x7368b1b30680?)
github.com/status-im/status-go/vendor/github.com/ethereum/go-ethereum/p2p/enode/node.go:89
github.com/status-im/status-go/vendor/github.com/waku-org/go-waku/waku/v2/peermanager.(*PeerManager).
AddDiscoveredPeer(0xc000e0e790, {0x4, {{0xc0004dd260, 0x27}, 
{0xc0006b5a00, 0x3, 0x4}}, 0x0, {0xc000f376b0, 0x1, ...}}, ...)
github.com/status-im/status-go/vendor/github.com/waku-org/go-waku/waku/v2/peermanager/peer_manager.go:432 +0x170
github.com/status-im/status-go/vendor/github.com/waku-org/go-waku/waku/v2/node.(*WakuNode).AddDiscoveredPeer(...)
github.com/status-im/status-go/vendor/github.com/waku-org/go-waku/waku/v2/node/wakunode2.go:710
github.com/status-im/status-go/wakuv2.(*Waku).connect(0xc0008fa5a0, {{0xc0004dd260, 0x27}, 
{0xc0006b5a00, 0x3, 0x4}}, 0x4)
github.com/status-im/status-go/wakuv2/waku.go:452 +0x1a6
created by github.com/status-im/status-go/wakuv2.(*Waku).discoverAndConnectPeers.func1
github.com/status-im/status-go/wakuv2/waku.go:415 +0x154
```

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.179.24",
-    "commit-sha1": "701b27632438a8328506cba23943bbdeb8b15b0c",
-    "src-sha256": "1jp3r6pb3q3jzjdlx8yl4js63q4av36n0wx9akb0pnarm7mwpfpb"
+    "version": "v0.179.25",
+    "commit-sha1": "0061c563f2d50679d812c5dc2978a96c49a93314",
+    "src-sha256": "0glzz5af873ym4qfbya14l38mjc8brkinpc7i57bywzsgydjqdp2"
 }

